Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml

```diff
@@ -8,6 +8,7 @@
   - type: HumanoidAppearance
     species: Vulpkanin
   - type: Hunger
+    baseDecayRate: 0.019166666659 # 15% more than default
```

Comment:
This for me personnaly is a nono.
I seen the effect of it and its horible. (on frontier)

Just please remove this.

Reply:
I disagree, I've also seen the effects of this and it was a rather interesting experience.

Reply:
@Mnemotechnician I reduced the 25% increased hunger rate to 15% due to feedback. What do you think is the ideal number for it?

I initially chose 25%, because EE does not have an increased hunger rate unlike Delta-V (see https://github.com/DeltaV-Station/Delta-v/pull/1302, https://github.com/DeltaV-Station/Delta-v/pull/1384). A 25% increase does not come even close to the default Delta-V hunger rate.

Reply:
@FoxxoTrystan I've done some research on the hunger rate changes of vulps on Frontier. It seems like they were set to a whopping **3x** hunger increase from the Frontier base hunger rate (`0.02f`).
![image](https://github.com/user-attachments/assets/f70e9593-81b8-4363-a24c-7d21a8934dd7)

That's a LOT and puts it in context for me. A 1.25x hunger increase will likely not feel the same at all as a drastic 3x hunger increase.

With the base hunger rate of ~0.0166, the fastest time to get to starving is around 72 minutes. With the hunger rate multiplied to 25%, it's still 58 minutes, still nearly an **hour** of playing. With that in mind, I'm reverting the hunger rate increase to 25%.

Reply:
> @Mnemotechnician I reduced the 25% increased hunger rate to 15% due to feedback. What do you think is the ideal number for it?

Oh, I meant the opposite - I agree with the change.

Reply:
okay then. we are sticking with 25%. if the global hunger rate gets increased like in DV, that's when i'm inclined to take another look at the hunger rate increase

Reply:
25% is still huge in my opinion, but we will see.

Honestly i do prefer the 15%